doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -78,8 +74,8 @@ ifdef::PAM[]
 +
 .. *Dialect*: Enter the Hibernate dialect for the server, depending on the server type:
 +
-*** `org.hibernate.dialect.MySQL5InnoDBDialect` (used for MySQL and MariaDB)
-*** `org.hibernate.dialect.PostgreSQL82Dialect`
+*** `org.hibernate.dialect.MySQL5InnoDBDialect` (used for MySQL version 5.7 and for MariaDB)
+*** `org.hibernate.dialect.MySQL8Dialect` (used for MySQL version 8)
```

Comment:
Did you remove the PostgreSQL82 dialect by mistake?

Reply:
Yes, that's a mistake, I will readd